website/docs/wallet/withdraw-validator.md

```diff
@@ -25,45 +25,25 @@ The **Capella/Shanghai Ethereum** upgrade lets you withdraw your validator nodes
 
 In this how-to, you'll learn how to perform both types of withdrawals.
 
+## In a nutshell
 
+Withdrawals are initiated by updating the `withdrawal_credentials` field on the validator with an Ethereum address of your choosing using your mnemonic. 
 
+**note: updating the `withdrawal_credentials` is irreversible, so you must carefully verify the Ethereum address you choose**
 
-## Before you begin
+We will be covering the following steps in this guide:
 
-Familiarity with the following resources and terms will help you confidently complete this guide:
-
-<!-- TODO: These terms can now be moved into Glossary CMS and embedded via quicklooks to further streamline the content experience - ping Mick if you'd like to help with this. -->
+- download [staking-deposit-cli](https://github.com/ethereum/staking-deposit-cli)
+- gather validator information for the [staking-deposit-cli](https://github.com/ethereum/staking-deposit-cli)
+- run the [staking-deposit-cli](https://github.com/ethereum/staking-deposit-cli) in an `offline` environment with your mnemonic to generate the `blstoexecutionchange` message(s)
```

Comment:
I wonder how much we should say specific terms in this outline such as blstoexecutionchange jargon. Is there a way in which we could say something like "signed withdrawal request"?